.github/workflows/hypertrace-ingester/docker-compose.yml

```diff
@@ -82,6 +82,8 @@ services:
       default:
         aliases:
           - pinot-controller
+          - pinot-server
```

Comment:
Heads up @kotharironak - the change in https://github.com/hypertrace/hypertrace-service/pull/55 effects the tests in a bunch of repos. Not sure if there's more to find, but may be worth a search.

Reply:
@aaron-steinfeld I had addressed this as part of this PR - https://github.com/hypertrace/hypertrace-ingester/pull/91, but, looks like it was not yet merged. I will scan other repos.